config/initializers/proxy.rb

```diff
@@ -1,9 +0,0 @@
-if ENV['PROXY_HOST']
```

Comment:
interesting, so I'm guessing that the need for a proxy wasn't needed anymore which called for this files deletion right? do the Proxy environment variables still exist as well?